beacon-chain/verification/filesystem.go

```diff
@@ -21,3 +23,32 @@ func VerifiedROBlobFromDisk(fs afero.Fs, root [32]byte, path string) (blocks.Ver
 	}
 	return blocks.NewVerifiedROBlob(ro), nil
 }
+
+func VerifiedRODataColumnFromDisk(file afero.File, root [fieldparams.RootLength]byte, sszEncodedDataColumnSidecarSize uint32) (blocks.VerifiedRODataColumn, error) {
```

Comment:
Missing godoc here. 

Take a look at the [live godocs page](https://pkg.go.dev/github.com/OffchainLabs/prysm/v6@v6.0.2/beacon-chain/verification) or run [godocs locally](https://pkg.go.dev/golang.org/x/tools/cmd/godoc)

Reply:
Fixed in b8fb77fc6fd7d35239602fd3b8cb292b54159397.